.github/workflows/publish-js-recon.yml

```
@@ -131,4 +131,20 @@ jobs:
+      - publish-docker
+      - publish-ghcr
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Merge main and dev
+        run: |
+          echo "Merging main into dev"
+          git checkout dev
+          git merge main
+          git push origin dev
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)